datasrc/network.py

```diff
@@ -403,11 +403,125 @@
 		NetStringStrict("m_Reason"),
 		NetBool("m_Force"),
 	]),
-	
+
 	### ModAPI
-	
+
 	NetMessage("ModAPI_Sv_Broadcast", [
 		NetStringStrict("m_pMessage"),
 	]),
-	
+
+
+	### NetGUI
+	## Server
+	NetMessage("ModAPI_Sv_GuiRemoveElement", [
+		NetIntAny("m_Type"),
+		NetIntAny("m_ID"),
+	]),
+
+	NetMessage("ModAPI_Sv_GuiRequestData", [
+		NetIntAny("m_ID"),
+		NetIntAny("m_Type"),
+	]),
+
+	NetMessage("ModAPI_Sv_GuiUIRect", [
+		NetIntAny("m_ID"),
+		NetArray(NetIntRange("m_Dimension", 0, 100), 4),
+		NetArray(NetIntRange("m_Color", 0, 100), 4),
+		NetIntRange("m_Corner", 0, 255),
+		NetIntRange("m_RoundingX10", 0, 500),
+	]),
+
+	NetMessage("ModAPI_Sv_GuiLabel", [
+		NetIntAny("m_ID"),
+		NetString("m_Text"),
+		NetArray(NetIntRange("m_Dimension", 0, 100), 4),
+		NetArray(NetIntRange("m_Color", 0, 100), 4),
+		NetIntAny("m_FontSize"),
+		NetIntRange("m_FontAlign", 0, 2),
+		NetIntRange("m_MaxTextWidth", 0, 100),		
+	]),
+
+	NetMessage("ModAPI_Sv_GuiButtonMenu", [
+		NetIntAny("m_ID"),
+		NetString("m_Text"),
+		NetIntRange("m_Selected", 0, 1),
+		NetArray(NetIntRange("m_Dimension", 0, 100), 4),
+	]),	
+
+	NetMessage("ModAPI_Sv_GuiEditBox", [
+		NetIntAny("m_ID"),
+		NetArray(NetIntRange("m_Dimension", 0, 100), 4),
+		NetString("m_Title"),
+		NetIntAny("m_SplitValue"),
+		NetIntAny("m_MaxTextWidth"),
+		NetIntRange("m_Password", 0, 1),
+	]),
+
+	NetMessage("ModAPI_Sv_GuiCheckBox", [
+		NetIntAny("m_ID"),
+		NetArray(NetIntRange("m_Dimension", 0, 100), 4),
+		NetString("m_Text"),
+		NetIntRange("m_Checked", 0, 1),
+	]),
+
+	NetMessage("ModAPI_Sv_GuiCheckBoxNumber", [
```

Comment:
I'm not sure if it is used because I don't play that much 0.7 yet ^^
But if you look in menus_ingame.cpp, where the netgui-ones are being rendered, I just used a existing vanilla function to render it. So it is at least possible in vanilla :D